modules/alerting/main.tf

```diff
@@ -0,0 +1,75 @@
+######################################################
+# SNS topics
+######################################################
+
+// This topic is used for deployment failure alerts etc
+resource "aws_sns_topic" "ecs_deployment_alerts" {
+  name = "${var.namespace}-${var.stage}-ecs-deployment-alerts"
+}
+
+
+######################################################
+# SNS topic subscriptions
+######################################################
+
+// This allows one or more webhooks to be subscribed to the ecs deployments alerts topic
+resource "aws_sns_topic_subscription" "ecs_deployment_alerts_webhook_subscription" {
```

Comment:
I am thinking we can be a bit less opinionated here on how the subscriptions should work, and instead just expose the SNS alert topics as outputs on our module. Customers can then add the `aws_sns_topic_subscription` resource outside of our stack